src/tools/md_blow.js

```diff
@@ -23,88 +26,98 @@ argv.key = argv.key || ('md_blow-' + Date.now().toString(36));
 
 main();
 
-function main() {
-    return client.create_auth_token({
+async function main() {
+    try {
+        await system_store.load();
+        await client.create_auth_token({
             email: argv.email,
             password: argv.password,
             system: argv.system,
-        })
-        .then(() => blow_objects())
-        .then(() => process.exit(0))
-        .catch(() => process.exit(1));
+        });
+        await blow_objects();
+        process.exit(0);
+    } catch (err) {
+        dbg.error('FATAL', err);
+        process.exit(1);
+    }
 }
 
-function blow_objects() {
+async function blow_objects() {
     let index = 0;
 
-    function blow_next() {
+    async function blow_next() {
         if (index >= argv.count) return;
         index += 1;
-        return blow_object(index).then(blow_next);
+        await blow_object(index);
+        await blow_next();
```

Comment:
Instead of async recursion we can use a simple loop:
```javascript
while (index < argv.count) {
    index += 1;
    await blow_object(index);
}
```